pkgs/applications/window-managers/dwm/dwm-status.nix

```diff
@@ -1,4 +1,9 @@
-{ stdenv, rustPlatform, fetchFromGitHub, dbus, gdk_pixbuf, libnotify, pkgconfig }:
+{ stdenv, rustPlatform, fetchFromGitHub, dbus, gdk_pixbuf, libnotify, pkgconfig, xorg, alsaUtils }:
+
+let
+  runtimeDeps = [ xorg.xsetroot ]
+    ++ lib.optional (alsaUtils != null) alsaUtils;
```

Comment:
As you can see in the ofborg failure you are missing an import to the lib attributeset:

```
undefined variable 'lib' at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/eval-0-gleber.ewr1.nix.ci/pkgs/applications/window-managers/dwm/dwm-status.nix:5:8
```